cgroups/fs/cpuset.go

```diff
@@ -48,6 +48,11 @@ func (s *CpusetGroup) GetStats(path string, stats *cgroups.Stats) error {
 }
 
 func (s *CpusetGroup) ApplyDir(dir string, cgroup *configs.Cgroup, pid int) error {
```

Comment:
Yeah, checking this outside would duplicate this logic, I like it here better too :+1: